connector/launchers/oaebudt-connector/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Stage 1: Build the JAR using Gradle
+FROM eclipse-temurin:23.0.2_7-jdk-alpine AS builder
```

Comment:
nit: java 24 has been released, we could consider to update our base images (not in this PR)